From 5def24d9e096467e702a805643b3b98f9e47cae7 Mon Sep 17 00:00:00 2001
From: Seppo Yli-Olli <seppo.yliolli@gmail.com>
Date: Sat, 8 Apr 2023 12:33:57 +0300
Subject: [PATCH] Remove SDL2 dependency for gstreamer-plugins-base

It should only be required by examples which we disable
---
 elements/components/gstreamer-plugins-base.bst | 1 -
 1 file changed, 1 deletion(-)

diff --git a/elements/components/gstreamer-plugins-base.bst b/elements/components/gstreamer-plugins-base.bst
index 9a5e13e90c..604c2f6d80 100644
--- a/elements/components/gstreamer-plugins-base.bst
+++ b/elements/components/gstreamer-plugins-base.bst
@@ -23,7 +23,6 @@ depends:
 - components/mesa-headers.bst
 - components/iso-codes.bst
 - components/alsa-lib.bst
-- components/sdl2.bst
 
 variables:
   command-subdir: subprojects/gst-plugins-base
-- 
GitLab

